deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -0,0 +1,278 @@
+        LogFilePrefix: s3/data-broker/
+
+  ManifestBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      AccessControl: PublicRead
+      LoggingConfiguration:
+        DestinationBucketName:
+          Fn::ImportValue: !Join [':', [!Ref InfrastructureStackName, 'LogBucket']]
+        LogFilePrefix: s3/data-broker/
+      WebsiteConfiguration:
+        IndexDocument: index.json
+        ErrorDocument: index.json
+      CorsConfiguration:
+        CorsRules:
+          - Id: DefaultCorsHeader
+            AllowedHeaders:
+              - Authorization
+            AllowedMethods:
+              - GET
+            AllowedOrigins:
+              - '*'
+            MaxAge: 3000
+
+  LambdaExecutionRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: "sts:AssumeRole"
+      Policies:
+        - PolicyName: ImageDerivativeCacheBucketPermissions
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              -
+                Action:
+                  - "s3:GetObject"
+                  - "s3:PutObject"
+                  - "s3:ListBucket"
+                  - "s3:ListBucketVersions"
+                  - "s3:PutObjectVersionAcl"
+                  - "s3:PutObjectAcl"
+                Effect: "Allow"
+                Resource:
+                  - !GetAtt ProcessBucket.Arn
+                  - Fn::Join:
+                      - ""
+                      -
+                        - !GetAtt ProcessBucket.Arn
+                        - "/*"
+                  - !GetAtt ManifestBucket.Arn
+                  - Fn::Join:
+                      - ""
+                      -
+                        - !GetAtt ManifestBucket.Arn
+                        - "/*"
+                  - Fn::Join:
+                    - ""
+                    -
+                      - !Sub "arn:aws:s3:::"
+                      - !Ref ImageSourceBucket
+                  - Fn::Join:
+                    - ""
+                    -
+                      - !Sub "arn:aws:s3:::"
+                      - !Ref ImageSourceBucket
+                      - "/*"
+
+
+  IndexInputLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: handler.run
+      Role: !GetAtt [ LambdaExecutionRole, Arn ]
+      Runtime: python3.7
+      Timeout: 25
+      CodeUri: json-from-csv/
+      Environment:
+        Variables:
+          EnvType: !Ref EnvType
+          PROCESS_BUCKET: !Ref ProcessBucket
+          MANIFEST_BUCKET: !Ref ManifestBucket
+          IMAGE_BUCKET: !Ref ImageSourceBucket
+          IMAGE_SERVER_URL: !Ref ImageServerUrl
+          MANIFEST_URL:
+            Fn::Join:
+              - ""
+              -
+                - !Ref ManifestBucket
+                - ".s3-website-us-east-1.amazonaws.com"
```

Comment:
I think this might be better to parameterize the region here since the bucket is created by this CFN. You can probably use the `AWS::Region`, or even better would be a `!GetAtt ManifestBucket.WebsiteURL` here.